be/src/services/vocabulary.service.ts

```diff
@@ -202,6 +235,66 @@ export class VocabularyService {
     }
     return {};
   }
+
+  private createMongoAggregationPipeline(param: {
```

Comment:
```suggestion
  private createMongoAggregationPipeline(matchIds?: string | ObjectId, filter?: Filter<Vocabulary>, options?: FindOptions): Document[] {
```
As far as I see you just dereference the argument object and use its properties, so why do you not use just three method arguments? The readability of the method signature is inhibited